static/src/javascripts/projects/admin/bootstraps/radiator.js

```diff
@@ -74,7 +74,18 @@ define([
             }
 
             function renderDeploys(stage, target) {
-                Object.keys(latestDeployments[stage]).forEach(function (deployment)  {
+                Object.keys(latestDeployments[stage])
```

Comment:
consider putting this in a named var and then using the var below in the foreach just to make it clear what we're getting out